src/storage_utils.cpp

```diff
@@ -148,11 +148,10 @@ namespace libtorrent { namespace aux {
 		// the number of bytes left to read in the current file (specified by
 		// file_index). This is the minimum of (file_size - file_offset) and
 		// bytes_left.
```

Comment:
you orphaned this comment now. It need to move along with the variable definition